sys/include/net/netdev_events.h

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
the old ng_netdev has the same events. I didn't want to copy them as long as both interfaces exist.